components/form/Toggle.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { Label, ErrorMessage } from '.';
+import classNames from 'classnames';
+
+const Toggle = ({ containerClassName, label, name, value, toggleClassName }) => {
+  const { register } = useFormContext();
+  return (
+    <>
+      <div className={classNames('relative flex items-center', containerClassName)}>
+        <input
+          name={name}
```

Comment:
From `register` API, name will be set by the `register(name)`, so I think it is safe to remove this to enhance consistency.

https://react-hook-form.com/api/useform/register

```suggestion

```